### ISSUE TITLE
Namespace is incorrectly marked as unused in a VB project

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#38304]
**Version Used**: 
Version 16.3.0 Preview 3.0 [29221.207.master]

**Steps to Reproduce**:

I tried to break this down to a small repro and failed.

1. `git clone http://github.com/dotnet/project-system`
2. `cd project-system`
3. `git checkout 0f64eeddeee286f73fabe7d68827b7adfa328d71`
4. `ProjectSystem.sln`
5. Open `src\Microsoft.VisualStudio.Editors\PropPages\CodeAnalysisPropPage.vb`

**Expected Behavior**:
No error or greying out of `Import System.Diagnostics`

**Actual Behavior**:
Greying out of `System.Diagnostics` and if you comment it out:

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	BC30456	'Assert' is not a member of 'Microsoft.VisualStudio.ProjectSystem.Debug'.	Microsoft.VisualStudio.Editors	C:\project-system\src\Microsoft.VisualStudio.Editors\PropPages\CodeAnalysisPropPage.vb	147	Active
Error	BC30456	'Assert' is not a member of 'Microsoft.VisualStudio.ProjectSystem.Debug'.	Microsoft.VisualStudio.Editors	C:\project-system\src\Microsoft.VisualStudio.Editors\PropPages\CodeAnalysisPropPage.vb	163	Active
Error	BC30456	'Assert' is not a member of 'Microsoft.VisualStudio.ProjectSystem.Debug'.	Microsoft.VisualStudio.Editors	C:\project-system\src\Microsoft.VisualStudio.Editors\PropPages\CodeAnalysisPropPage.vb	164	Active
Error	BC30456	'Assert' is not a member of 'Microsoft.VisualStudio.ProjectSystem.Debug'.	Microsoft.VisualStudio.Editors	C:\project-system\src\Microsoft.VisualStudio.Editors\PropPages\CodeAnalysisPropPage.vb	180	Active

```

If you remove the `Imports Microsoft.VisualStudio.ProjectSystem` import from lower in the imports, then the build errors goes away.
